### DISCUSSION TITLE
feat : `gpt4free` library integration with `ollama-webui`

### DISCUSSION THREAD

**Discussion by hemangjoshi37a** [open-webui/open-webui#591] [General]
here the [https://github.com/xtekky/gpt4free](https://github.com/xtekky/gpt4free) library provides premium paid models for free by exploiting its API endpoints. How can we take advantage of this and can use inside ollama-webui.

Please some dev help to implement this .

**Comment by tjbck** [comment]
Seem like it's out of scope for this project, I'll move this to discussion, Thanks!

**Comment by justinh-rahb** [comment]
Sounds like a great way to get IP-banned.

**Comment by DxrkFuture** [comment]
I wish there was some utility in the “workspace” to be able to add “provider” to API requests in json and change it from your list of providers in the settings
